project-base/app/Bootstrap.php

```diff
@@ -53,7 +54,11 @@ public function run()
         }
 
         $kernel = new AppKernel($this->environment, EnvironmentType::isDebug($this->environment));
-        Request::setTrustedProxies(['127.0.0.1'], Request::HEADER_X_FORWARDED_ALL);
+
+        $parameters = $this->loadParametersConfigFile($kernel);
```

Comment:
Not taking the parameters from the actual DIC may lead to unforeseen problems. Did you consider booting the Kernel manually via `$kernel->boot()` and using `$kernel->getContainer()->getParameter('trusted_proxies'))` instead?

Reply:
Oh I didnt think of `boot()` function, I was still getting null error. Thats definitely much better solution :)